fairlearn/widget/fairlearnDashboard.py

```diff
@@ -25,7 +25,7 @@ class FairlearnDashboard(object):
     def __init__(
             self, *,
             sensitive_features,
-            true_y, predicted_ys,
+            y_true, predicted_ys,
```

Comment:
Should that be `ys_pred` as well?

Reply:
I asked Miro for his opinion, going with y_pred singular, and also accepting a dictionary as input, mapping it to a 2D array and an matching 1D array of model names